src/genetic_testing/assay_target/primer_design.py

```diff
@@ -0,0 +1,183 @@
+import re
+from copy import deepcopy
+from io import StringIO
+
+import numpy as np
+import pandas as pd
+from Bio import SeqIO
+from fuzzysearch import find_near_matches
+
+from genetic_testing.assay_target.datatypes import AssayTargetColumns
+
+
+class primer_list:
+    def __init__(self, primer_seq, mean_score):
+        """
+
+        :rtype: object
+        """
+        self.primer_seq = primer_seq
+        self.mean_score = mean_score
+
+
+class exact_match_primer:
+    def __init__(self, primer_seq, window_l, n_match):
+        self.primer_seq = primer_seq
+        self.window_l = window_l
+        self.n_match = n_match
+
+
+def import_files(target_files, off_target_files, reference_sequence):
+    target_files_sorted = sorted(target_files, key=lambda x: x.name)
+    off_target_files_sorted = sorted(off_target_files, key=lambda x: x.name)
+    # targ_fasta_paths = glob.glob(os.path.join(target_path, "*.fasta"))
+    # off_fasta_paths = glob.glob(os.path.join(off_path, "*.fasta"))
+
+    # create lists of sequences (target and off target)
+    # and labels (0 = target, !0 = off target)
+    aln = []
+    labs = []
+    index = -1  # default value for the index of the reference sequence
+    for fasta_file in target_files_sorted:
+        print(fasta_file.name)
```

Comment:
can we use a logger instead of print? Where do prints go in streamlit?

Reply:
Yes, good catch. I will remove this print statement. I have a logger module but I will not add anything here to keep changes to minimum.